com.github.hneemann.digital/build.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+NAME=com.github.hneemann.digital
+ARCH=x86_64
+
+buildonly=0
+while getopts ba: opt 2>/dev/null 1>&2
+do
+case ${opt} in
+    "b") buildonly=1;;
+    "a") ARCH=${OPTARG};;
+    ?) echo "wrong parameters"; exit 0;;
+esac
+done
+
+if [ $# -ge 1 ]; then
+ARCH=${1}
+echo registering qemu-user-static
+sudo podman run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

Comment:
this will never work inside flatpak.

Also this file is never referenced by the manifest so I don't this it's purpose.